EIPS/eip-2935.md

```diff
@@ -26,22 +27,34 @@ Additional secondary motivations include:
 
 | Parameter | Value |
 | - | - |
-| `FORK_BLKNUM` | TBD |
+| `FORK_TIMESTAMP` | TBD |
 | `HISTORY_STORAGE_ADDRESS` | `0xfffffffffffffffffffffffffffffffffffffffe`|
+| `HISTORY_SERVE_WINDOW` | `256` |
 
-At the start of processing any block where `block.number > FORK_BLKNUM` (ie. before processing any transactions), run `sstore(HISTORY_STORAGE_ADDRESS, block.number - 1, block.prevhash)`.
+At the start of processing any block where `block.timestamp > FORK_TIMESTAMP` (ie. before processing any transactions):
 
-When `block.number > FORK_BLKNUM + 256`, change the logic of the `BLOCKHASH` opcode as follows: if `FORK_BLKNUM <= arg < block.number`, return `sload(HISTORY_STORAGE_ADDRESS, arg)`. Otherwise return 0.
+ * run `sstore(HISTORY_STORAGE_ADDRESS, block.number - 1, block.prevhash)`. 
```

Comment:
Nit: there is currently no note on how to avoid EIP-158 state clearing. If an account has zero balance, zero nonce, no code, but storage, it is still considered dead, so any touch-action on this contract will delete the entire state. (So either pre-fund this account, or bump the nonce to one, this should be safe since no one should be able to deploy a contract here / have the keys of this contract)